app/assets/javascripts/app/discussions/index.html.slim

```diff
@@ -2,10 +2,11 @@ div ui-view="sidebar"
 
 .TopBarLayout
   .TopBarLayout-header
-    md-toolbar.md-whiteframe-z1.Toolbar
```

Comment:
In this case, the Toolbar class was only used to give color, this isn't needed anymore because all the colors are controlled by the themeProvider. 
I also removed the rules regarding background color and color from it due to their not being used anywhere else.